Doc/library/pathlib.rst

```diff
@@ -545,6 +547,30 @@ Pure paths provide the following methods and properties:
       PureWindowsPath('c:/Program Files')
 
 
+.. method:: PurePath.makepath(*other)
```

Comment:
Should be plural, I think, similar to the way we have `*args` rather than `*arg`:

```suggestion
.. method:: PurePath.makepath(*others)
```

Reply:
I've switched it to `*pathsegments`, matching the documentation for the initialiser. Is that any good?

Reply:
it's good!